app/README.md

```diff
@@ -1,5 +1,5 @@
-Pour exécuter l'application Streamlit de , lancer la commande suivante depuis ce répertoire : 
-    `streamlit run odi_streamlit.py`
+Pour exécuter l'application Streamlit de , lancer la commande suivante depuis le répertoire parent : 
```

Comment:
Would be great to have an invoke task for this, something like `invoke streamlit`.

Reply:
Thanks. Invoke command added.

Reply:
we'll fix the encoding issue in  a later PR.  We still have some work to do on the data.